pwiz_tools/Skyline/Model/Lib/BlibData/DbRefSpectraPeakAnnotations.cs

```diff
@@ -78,6 +78,10 @@ public static ICollection<DbRefSpectraPeakAnnotations> Create(DbRefSpectra refSp
             return resultList;
         }
 
+        public override string ToString()
+        {
+            return (Name??string.Empty)+@" i"+PeakIndex; // For debugging convenience

```

Comment:
string.Empty seems like a strange choice here. Is that really what will be shown for this annotation? I would think something like "<unknown>" or "<unnamed>". The spacing is also nonstandard here. It is one thing to bang this in for your own debugging and another to commit it to the project for the benefit of others.

Reply:
Fair enough, but what *is* our standard for this kind of "user never sees it, but just showing a class name in the debugger isn't at all helpful to developers" formatting? If there's a right way to do it I'm happy to go that route. I do think it's worth leaving in. 

But I can certainly leave it out if there's a philosophical objection to it, particularly if I'm remaking this as a commit to 19.2 with cherry-pick back to master.